mon-pix/app/components/dashboard/content.js

```diff
@@ -1,7 +1,22 @@
 import Component from '@glimmer/component';
+import orderBy from 'lodash/orderBy';
 
 export default class Content extends Component {
   get hasCampaignParticipationOverviews() {
-    return this.args.model.length > 0;
+    const campaignParticipationOverviews = this.args.model.campaignParticipationOverviews;
+    return campaignParticipationOverviews && campaignParticipationOverviews.length > 0;
+  }
+
+  get hasRecommendedCompetences() {
+    const scorecards = this.args.model.scorecards;
+    return scorecards && scorecards.length > 0;
```

Comment:
Proposition : 
```suggestion
    return this. recommendedScorecards.length > 0;
```

J'ai l'impression que sinon, on pourrait avoir 0 compétences non commencé mais que la méthode `hasRecommendedCompetences` renvoie true